Source/WebKit/WebProcess/WebCoreSupport/WebFileSystemStorageConnection.cpp

```diff
@@ -156,7 +156,7 @@ void WebFileSystemStorageConnection::createSyncAccessHandle(WebCore::FileSystemH
 void WebFileSystemStorageConnection::closeSyncAccessHandle(WebCore::FileSystemHandleIdentifier identifier, WebCore::FileSystemSyncAccessHandleIdentifier accessHandleIdentifier, FileSystemStorageConnection::VoidCallback&& completionHandler)
 {
     if (!m_connection)
-        return;
+        return completionHandler(WebCore::Exception { WebCore::UnknownError, "Connection is lost"_s });
 
     m_connection->sendWithAsyncReply(Messages::NetworkStorageManager::CloseSyncAccessHandle(identifier, accessHandleIdentifier), [completionHandler = WTFMove(completionHandler)]() mutable {
```

Comment:
Could probably be rewritten: `sendWithAsyncReply(..., WTFMove(completionHandler));`